### PR TITLE
Add Benthos configs for BOM forwarding

### PR DESCRIPTION
**Pull request by nscuro**

### CODE REVIEW

File: scripts/benthos/bom-forward_hyades-to-hyades.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (mehab) [inline scripts/benthos/bom-forward_hyades-to-hyades.yml:1]: this one is also going to be bom-forward_vanilla-to-hyades right?

**Review by mehab — APPROVED** [review]
lgtm

File: scripts/benthos/bom-forward_hyades-to-hyades.yml (hunk 1 above)
Comment (nscuro) [reply scripts/benthos/bom-forward_hyades-to-hyades.yml:1]: Not sure I understand, could you elaborate?

This file is specifically for forwarding *from* Hyades *to* another Hyades instance.

In contrast to the other file `bom-forward_vanilla-to-hyades.yml`, in this one we have to fetch the BOM from the API server, whereas vanilla DT sends the original BOM as part of the notification.

Comment (mehab) [reply scripts/benthos/bom-forward_hyades-to-hyades.yml:1]: okay then my mistake. looks good :)

**Pull request merged**